### PR TITLE
docs: Document the shared cache

### PR DESCRIPTION
**Pull request by flub**
Some basic pointers, this is an advanced thing after all.

NATIVE-248

#skip-changelog

### CODE REVIEW

**Comment by loewenheim** [comment]
Did you forget the file?

**Comment by flub** [comment]
> Did you forget the file?

lol, of course i did

File: docs/advanced/shared-cache.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
+# Shared Cache
+
+When running multiple symbolicators each instance has their own local
```
Comment (loewenheim) [inline docs/advanced/shared-cache.md:3]:
```suggestion
When running multiple symbolicators each instance has its own local
```

File: docs/advanced/shared-cache.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+cache.  Starting up a new symbolicator instance in such an environment
+means it will have a cold cache and take a long time before it can
+process events at the same rate.  To remedy this symbolicators can be
+configured to share their caches.
+
+When configured like this all caches will be shared, both original
+files and derived caches, thus a new symbolicator will not need to do
+all the computation to create missing common derrived caches.  However
```
Comment (loewenheim) [inline docs/advanced/shared-cache.md:11]:
```suggestion
all the computation to create missing common derived caches.  However
```

**Review by loewenheim — APPROVED** [review]

**Review by Swatinem — APPROVED** [review]

**Pull request merged**